inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/config.cpp

```diff
@@ -84,13 +84,6 @@ namespace {
             ::testing::ValuesIn(multiconf)),
             CorrectConfigAPITests::getTestCaseName);
 
-    INSTANTIATE_TEST_SUITE_P(smoke_Auto_BehaviorTests, CorrectConfigAPITests,
```

Comment:
we have similar duplication above for the CorrectConfigTests, but why here we removed the AUTO instance for the CorrectConfigAPITests?

Reply:
add it and fixed